modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryLocalServiceImpl.java

```diff
@@ -241,6 +249,39 @@ GroupConstants.DEFAULT_LIVE_GROUP_ID, getLocalizationMap(name),
 		return accountEntry;
 	}
 
+	@Override
+	public AccountEntry addIncompleteAccountEntry(
```

Comment:
I think at some point we will need to autogenerate these methods. I think it's OK to have them manually for now until we have at least another use case and we can lift the pattern to service builder.